rtp_h264_extractor.lua

```diff
@@ -184,19 +188,20 @@ do
             local payloadTable = { h264_data() }
             local seqTable = { rtp_seq() }
             
-            if (#payloadTable) ~= (#seqTable) then 
-                log("ERROR: payloadTable size is "..tostring(#payloadTable)..", seqTable size is "..tostring(#seqTable))
-                return
-            end
+--            if (#payloadTable) ~= (#seqTable) then 
+--                log("ERROR: payloadTable size is "..tostring(#payloadTable)..", seqTable size is "..tostring(#seqTable))
+--                return
+--            end
```

Comment:
those lines is target to handle the case:  RTP packet, but payload is not h264, then this packet will be ignored.   You can't remove them simply.